homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -147,31 +167,135 @@ class XiaomiMiioSensorDescription(SensorEntityDescription):
         icon="mdi:fast-forward",
         state_class=STATE_CLASS_MEASUREMENT,
     ),
+    ATTR_MOTOR2_SPEED: XiaomiMiioSensorDescription(
+        key=ATTR_MOTOR2_SPEED,
+        name="Second Motor Speed",
+        native_unit_of_measurement="rpm",
+        icon="mdi:fast-forward",
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
     ATTR_ILLUMINANCE: XiaomiMiioSensorDescription(
         key=ATTR_ILLUMINANCE,
         name="Illuminance",
         native_unit_of_measurement=UNIT_LUMEN,
         device_class=DEVICE_CLASS_ILLUMINANCE,
         state_class=STATE_CLASS_MEASUREMENT,
     ),
-    ATTR_AIR_QUALITY: XiaomiMiioSensorDescription(
-        key=ATTR_AIR_QUALITY,
-        native_unit_of_measurement="AQI",
-        icon="mdi:cloud",
+    ATTR_ILLUMINANCE_LUX: XiaomiMiioSensorDescription(
+        key=ATTR_ILLUMINANCE,
+        name="Illuminance",
+        native_unit_of_measurement=LIGHT_LUX,
+        device_class=DEVICE_CLASS_ILLUMINANCE,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ATTR_AQI: XiaomiMiioSensorDescription(
+        key=ATTR_AQI,
+        name="Air Quality Index",
```

Comment:
I just noticed that in the Xiaomi Home app this value is specified as PM2.5 and not AQI. I wonder if we should not treat it as a PM2.5 sensor.

![Screenshot_20210813-124553__01](https://user-images.githubusercontent.com/478555/129346973-f1b7ee15-2c15-4e6d-b76e-025d8fbc66ad.jpg)